migration/migration.go

```diff
@@ -237,8 +238,14 @@ func uploadResources(config UploadBinariesConfig) error {
 		if err != nil {
 			return errors.Trace(err)
 		}
-		// TODO(menn0) - charmstore revision
-		// TODO(menn0) - unit revisions
+		for unitName, unitRev := range res.UnitRevisions {
```

Comment:
Because you can't set the unit revisions until the related app revision is there so it has to be done here.